src/com/epam/javacourse/hometask3/Bookstore.java

```diff
@@ -0,0 +1,13 @@
+package com.epam.javacourse.hometask3;
+
+public abstract class Bookstore {
+    public static void displayBooksByAuthor(String authorName, Book[] books) {
```

Comment:
You can declare this method without `static` as soon as your class is not abstract.
And instantiate it as `new Bookstore().displayBooksByAuthor(...)`